pandas/io/parquet.py

```diff
@@ -76,9 +76,9 @@ def write(self, df, path, compression='snappy',
                 table, path, compression=compression,
                 coerce_timestamps=coerce_timestamps, **kwargs)
 
-    def read(self, path):
+    def read(self, path, columns=None):
         path, _, _ = get_filepath_or_buffer(path)
-        return self.api.parquet.read_table(path).to_pandas()
+        return self.api.parquet.read_table(path, columns=columns).to_pandas()
```

Comment:
i’d like to pass thru kwargs as well; these won’t be specific names args just pass thru to the engine to validate
for both fp and pyarrow 
could just be a simple test with row_groups

Reply:
Ok, i think it is good to pass explicit options like columns which are supported by both backends and also pass the kwargs to be able to provide additional engine specific args.

Have to look at the test case.

Reply:
ok that’s fine
really want row_group support :) (next PR!)
also if u want: https://github.com/pandas-dev/pandas/issues/17102